jujugui/static/gui/src/app/components/env-list/env-list.js

```diff
@@ -47,6 +47,8 @@ YUI.add('env-list', function() {
         var envName = env.name || env.path;
         envs.push(
           <li className="env-list__environment"
+            role="menuitem"
+            tabIndex="-1"
```

Comment:
As we discovered this is correct as the user should use the arrow keys to navigate not tab.